src/runtime/offers-api.js

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
```

Comment:
Note: Only necessary during the TypeScript migration. Once the `src/runtime/offers-api.js` file is TypeScript, we can remove this eslint comment